api/src/main/java/ca/bc/gov/educ/api/graduation/config/RequestInterceptor.java

```diff
@@ -43,6 +44,18 @@ public boolean preHandle(HttpServletRequest request, HttpServletResponse respons
 		if (correlationID != null) {
 			ThreadLocalStateUtil.setCorrelationID(correlationID);
 		}
+		else {
+			ThreadLocalStateUtil.setCorrelationID(new UUID(1,1).toString());
+		}
+
+		// Header userName
+		val headerUserName = request.getHeader(EducGraduationApiConstants.HEADER_USER_NAME);
+		if (headerUserName != null) {
```

Comment:
If there is no username header, then check the JWT security token as on line 62 for username as a fallback. Otherwise username will be null.